### PR TITLE
Fixes #8301 - Add forward/reverse dns check

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
Adding check based on @ekohl suggestion from https://github.com/Katello/katello-installer/pull/522, proof it works in that PR

File: checks/dns.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Check to verify forward / reverse dns matches hostname
+
+def error_exit(message, code)
+  $stderr.puts message
+  exit code
+end
+
+@hostname = `hostname -f`.chomp
+@ip = `getent ahosts #{@hostname} | awk '/STREAM/ { print $1 }'`.chomp
+@reverse = `getent hosts #{@ip} | awk '{ print $2 }'`.chomp
```
Comment (ekohl) [inline checks/dns.rb]: This breaks when there are multiple IPs.

**Review by ekohl — CHANGES_REQUESTED** [review]
I should have verified this when approving the katello-installer PR. Other than that you can use closes #8301.

**Comment by chris1984** [comment]
@ekohl changed to yours, after testing today, can I get another review/merge please?

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Oh, if you could use ```Fixes #8301``` that'd make @theforeman-bot happy.

**Comment by chris1984** [comment]
@ekohl can i get a merge please, i fixed the commit message

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
tested standalone with/etc/hosts entires, seems to work

**Comment by mmoll** [comment]
merged, thanks @chris1984!

**Pull request merged**